lib/matplotlib/widgets.py

```diff
@@ -1007,7 +1007,12 @@ def __init__(self, ax, labels, active=0, activecolor='blue'):
             The index of the initially selected button.
         activecolor : color
             The color of the selected button.
+        direction : str
```

Comment:
Please use `orientation` instead of `direction` to be consistent with e.g `Slider`. Also the explicit long forms 'vertical', 'horizontal' are sufficient. Radio buttons are not used that often to justify redundant abbreviations 'v', 'h'.

Reply:
OK, done!